packages/discussions/README.md

```diff
@@ -0,0 +1,72 @@
+[![npm version][npm-img]][npm-url]
+[![build status][travis-img]][travis-url]
+[![gzip bundle size][gzip-image]][npm-url]
+[![Coverage Status][coverage-img]][coverage-url]
+[![apache licensed](https://img.shields.io/badge/license-Apache-green.svg?style=flat-square)](https://raw.githubusercontent.com/Esri/hub.js/master/LICENSE)
+
+[npm-img]: https://img.shields.io/npm/v/@esri/hub-discussions.svg?style=flat-square
+[npm-url]: https://www.npmjs.com/package/@esri/hub-discussions
+[travis-img]: https://img.shields.io/travis/Esri/hub.js/master.svg?style=flat-square
+[travis-url]: https://travis-ci.org/Esri/hub.js
+[gzip-image]: https://img.badgesize.io/https://unpkg.com/@esri/hub-discussions/dist/umd/discussions.umd.min.js?compression=gzip
+[coverage-img]: https://codecov.io/gh/Esri/hub.js/branch/master/graph/badge.svg
+[coverage-url]: https://codecov.io/gh/Esri/hub.js
+
+# THIS PACKAGE IS ACTIVELY UNDER DEVELOPMENT AND IS NOT TO BE USED IN A PRODUCTION ENVIRONMENT
+# @esri/hub-discussions
+
+> Module to interact with ArcGIS Hub Discussions API in Node.js and modern browsers.
+
+### Example
+
+```bash
```

Comment:
@cpgruber  can you add some more examples of using the API with specifics usages in the response. We are trying out [RDD](https://tom.preston-werner.com/2010/08/23/readme-driven-development.html) to design/update the rest of Hub.js to consider the DX (Developer Experience) making it easy to pickup + use the library by Hub + non-Hub engineers.

/cc @tomwayson @dbouwman @rweber-esri

Reply:
Examples in the function jsdoc will appear in the doc w/ the function... here so it's a great way to show simple usage...
![image](https://user-images.githubusercontent.com/119129/115734768-31e48f00-a347-11eb-97de-4478cef146d7.png)

Another good idea is to add a Guide into the docs...  guides are a great place to show how to do common things... i.e. 
![image](https://user-images.githubusercontent.com/119129/115734906-4c1e6d00-a347-11eb-8438-e3411e8f94e4.png)

To add a Guide, add a markdown doc in `/docs/src/guides`... at the top of that file, add an info block like

```
---
title: Working with Sites and Pages.
navTitle: Sites and Pages
description: Learn how to work with Sites, Pages and Domains using @esri/hub.js.
order: 20
group: 2-concepts
---
```

followed by the content.

Then run `npm run docs:serve` and it will fire up the doc site w/ a watcher - I don't think it will live-reload, but it will rebuild the doc a you save files